### ISSUE TITLE
The page is way too wide for calendar

### ISSUE THREAD

**Issue by ig-ealbert** [IgniteUI/indigo-design-docfx#76]
http://staging.infragistics.local/indigo-docfx/components/calendar.html

The width is MUCH WIDER than other pages and forces you to scroll!

**Comment by tiliev** [comment]
This is because of the images in the table:
```
| Do                              | Don't                             |
| ------------------------------- | --------------------------------- |
| ![](../images/calendar_do1.png) | ![](../images/calendar_dont1.png) |
| ![](../images/calendar_do2.png) | ![](../images/calendar_dont2.png) |
```
They appear to be to wide. Will take a look at this issue once #25 is closed.

**Comment by ig-ealbert** [comment]
This is fixed in the latest build.  I think it was resolved in #88.
